### PR TITLE
Implemented Undo and Redo feature

### PR DESCRIPTION
**Pull request by whyCaiJi**

### CODE REVIEW

**Review by nuslarry — APPROVED** [review]
gd

File: src/main/java/seedu/task/logic/commands/RedoCommand.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import seedu.task.logic.commands.exceptions.CommandException;
+
+/**
+ * Redo the previous change undone
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+    public static final String MESSAGE_SUCCESS = "Preivous undo restored.";
```
Comment (tylerrocha) [inline src/main/java/seedu/task/logic/commands/RedoCommand.java:13]: Typo here :)

**Review by tylerrocha — APPROVED** [review]
Nice work!
In the future do we want to allow commands such as complete and prioritize to be undoable?

**Comment by jay500s** [comment]
Looks good. We must consider how to redo and undo commands that involve recurring tasks next. This will be tricky because we use the Abstraction Occurrence Pattern is used and we will need a way to keep track of which RecurringOccurence to refer to in the stack.

**Review by jay500s — COMMENTED** [review]
Looks good. We must consider how to redo and undo commands that involve recurring tasks next. This will be tricky because we use the Abstraction Occurrence Pattern is used and we will need a way to keep track of which RecurringOccurence to refer to in the stack.

**Pull request merged**